lib/mods-view.js

```diff
@@ -62,7 +62,9 @@ ModsView.prototype.show = function (items, action, excludePaths) {
     }
     this.panel.show();
     this.setItems(items.filter(function (i) {
-        return excludePaths.indexOf(i.path) === -1;
+        return !excludePaths.some(function(pathToExclude) {
```

Comment:
Again just for testing now, but the idea is that if the path contains any of the strings in the exclude list, exclude it.